templates/bundles/SyliusShopBundle/Homepage/Block/_latestProductsCarousel.html.twig

```diff
@@ -0,0 +1,4 @@
+<h2 class="ui center aligned huge header">{{ 'sylius.homepage.brace_yoursel_winter_is_coming'|trans }}</h2>
+<div class="ui hidden divider"></div>
+
+{{ render(path('bitbag_sylius_cms_plugin_shop_block_render', {'code' : 'seasonal_promotion', 'template' : '@SyliusShop/Homepage/_carousel.html.twig'})) }}
```

Comment:
IMHO 
```suggestion
{{ render(path('bitbag_sylius_cms_plugin_shop_block_render', {'code' : 'seasonal_promotion', 'template' : 'Shop/Homepage/Blocks/_carousel.html.twig'})) }}
```

and the file itself should be placed in `templates/Shop/Homepage/Blocks/_carousel.html.twig`